code/modules/awaymissions/mission_code/wildwest.dm

```diff
@@ -135,9 +135,6 @@
 	var/triggerproc = "triggerrad1" //name of the proc thats called when the mine is triggered
 	var/triggered = 0
 
-/obj/effect/meatgrinder/New()
-	icon_state = "blobpod"
```

Comment:
We had a fair few `/New()` instances which just set vars to static vars which were already set

Reply:
`var/triggered = 0`

This is a boolean. Please change to FALSE/TRUE to please the nerd.

Reply:
Not happening, the whole point of this PR is just to add parent calls, not refactor half the files.